Sasquatch/SasquatchObjC/Info.plist

```diff
@@ -35,6 +35,10 @@
 	<true/>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Select photo for attachment</string>
+	<key>NSLocationAlwaysUsageDescription</key>
+	<string>The app needs to know your location to override the country code.</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
```

Comment:
You also need to add this one to the Swift app's Info.plist: https://github.com/Microsoft/AppCenter-SDK-Apple/blob/develop/Sasquatch/SasquatchSwift/Info.plist

Reply:
Updated